src/currencyviewer/widget/ndeepcurrencyviewer.js

```diff
@@ -23,31 +29,30 @@ mendix.widget.declare("CurrencyViewer.widget.ndeepcurrencyviewer", {
     },
     applyContext : function(context, callback){
         if (context)
-            mx.processor.getObject(context.getActiveGUID(), dojo.hitch(this, this.checkForPath));
+            mx.processor.getObject(context.getActiveGUID(), lang.hitch(this, this.checkForPath));

```

Comment:
`mx.processor.getObject` is probably depricated

Reply:
Fixed